src/matrixHelpers/formioScript/index.dev.js

```diff
@@ -0,0 +1,42 @@
+import { createScripts } from "./createScripts";
```

Comment:
make a index.test and index.staging (which points to beta-static)

We may gen a staging-static in the near future which is a mirror of beta to reduce confusion.